### ISSUE TITLE
Use OpenGL directly along with Andorra

### ISSUE THREAD

**Issue by z505** [astoeckel/andorra#2]
Is it possible to use OpenGL directly (api calls) if something was missing in Andorra that's not yet possible?

Or would it conflict with Andorra and not work so well.

If it was possible, this would be a way to exceed any limitations of Andorra... which would be really neat.

As I am a newbie to Andorra, I do not know the code well and what potential issues would arise or if this even possible.

**Comment by astoeckel** [comment]
Yes, given the statefullness of OpenGL this definitively works, but is not a good programming practice, as it ignores the intricate layers of abstraction provided by Andorra. The 3D interface abstraction layer provided by Andorra is actually quite powerful, you can basically build a 3D engine on top of it. So depending on what you're trying to do, the Andorra API abstraction layer might not be the limitation.
